publication-commons/src/test/java/no/unit/nva/publication/service/impl/ResourceServiceTest.java

```diff
@@ -1333,6 +1337,111 @@ void shouldSetAllTicketsToNotApplicableWhenMarkingDraftPublicationAsDraftForDele
         assertEquals(TicketStatus.NOT_APPLICABLE,ticket.fetch(ticketService).getStatus());
     }
 
+    @Test
```

Comment:
These tests all appear to duplicate the same setup, could it be extracted to a common util method?